rpcs3/rpcs3qt/infinity_dialog.cpp

```diff
@@ -0,0 +1,831 @@
+#include "stdafx.h"
+#include "Utilities/File.h"
+#include "Crypto/md5.h"
+#include "Crypto/aes.h"
+#include "Crypto/sha1.h"
+#include "infinity_dialog.h"
+#include "Emu/Io/Infinity.h"
+
+#include "util/asm.hpp"
+
+#include <locale>
+
+#include <QLabel>
+#include <QGroupBox>
+#include <QFileDialog>
+#include <QVBoxLayout>
+#include <QMessageBox>
+#include <QComboBox>
+#include <QPushButton>
+#include <QStringList>
+#include <QCompleter>
+
+infinity_dialog* infinity_dialog::inst = nullptr;
+std::array<std::optional<u32>, 7> infinity_dialog::figure_slots = {};
+static QString s_last_figure_path;
```

Comment:
I meant static member of the class

Reply:
It needs to be referenced in the create dialog and main dialog (2 separate classes), figured a global static string would suit here?